warp/handlers/signature_request_test.go

```diff
@@ -21,6 +22,14 @@ import (
 )
 
 func TestMessageSignatureHandler(t *testing.T) {
+	metricsEnabled := metrics.Enabled
+	if !metricsEnabled {
+		metrics.Enabled = true
+		t.Cleanup(func() {
+			metrics.Enabled = false
+		})
+	}
```

Comment:
This might not be needed given the `init()` block setting the metrics to Enabled, but I preferred to keep it here if we one day revert the default Metrics Enabled to false (as it is in geth)

Reply:
Could we put this code in a different location such as `plugin/evm/metricstest`
then have `metricstest.WithMetrics(t)` for example?

I'm assuming this code was added only to tests that were failing when metrics was disabled?

Reply:
Done in `plugin/evm/testutils/metrics.go` with a function `WithMetrics(t)`. I went a bit crazy on the use-testutils-only-for-tests-outside-plugin/evm (inspired by your import test + arran's check for test files), since I think it would be good to only have a `testutils` to reduce changes in geth code, not for our own code (prefer copy pasting and defining local code for readability + reduce package dependencies)